api/src/test/java/io/opentelemetry/trace/DefaultTracerTest.java

```diff
@@ -123,4 +119,33 @@ public void testSpanContextPropagationCurrentSpan() {
       scope.close();
     }
   }
+
+  @Test
+  public void testSpanContextPropagationCurrentSpanContext() {
+    Context context = TracingContextUtils.withSpanContext(spanContext, Context.current());
+    Scope scope = ContextUtils.withScopedContext(context);
+    try {
+      Span span = defaultTracer.spanBuilder(SPAN_NAME).startSpan();
+      assertThat(span.getContext()).isSameInstanceAs(spanContext);
+    } finally {
+      scope.close();
+    }
+  }
+
+  @Test
+  public void testSpanContextPropagationCurrentContextValues() {
+    Context context =
+        TracingContextUtils.withSpanContext(
+            SpanContext.create(
+                new TraceId(1, 1), new SpanId(1), TraceFlags.getDefault(), TraceState.getDefault()),
+            TracingContextUtils.withSpan(new DefaultSpan(spanContext), Context.current()));
+    Scope scope = io.opentelemetry.context.ContextUtils.withScopedContext(context);
```

Comment:
```suggestion
    Scope scope = ContextUtils.withScopedContext(context);
```

Reply:
Please fix.

Reply:
Please fix?

Reply:
This change is not required, as this was a test related to having both `SpanContext` and `Span` in `Context`, and thus the test itself was removed ;)